src/pcm/SoxrResampler.cxx

```diff
@@ -22,12 +22,30 @@
 #include "config/Block.hxx"
 #include "util/RuntimeError.hxx"
 #include "util/Domain.hxx"
+#include "util/Alloc.hxx"
+
 #include "Log.hxx"
 
 #include <soxr.h>
 
 #include <assert.h>
 #include <string.h>
+#include <math.h>
+
+struct soxr {
```

Comment:
Bad name. What does `soxr` mean?
Why not use `struct soxr_quality_spec`, which is essentially the same?